code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4060,3 +4060,14 @@
 			if(counter == available_snacks.len)
 				counter = 0
 			counter++
+
+
+/obj/item/weapon/reagent_containers/food/snacks/croissant
+	name = "croissant"
+	desc = "True french quisine."
+	icon_state = "croissant"
+
+	New()
```

Comment:
Bad practice. Replace this with 
/obj/item/weapon/reagent_containers/food/snacks/croissant/New()